go-client/TestSetupAbstraction_test.go

```diff
@@ -184,12 +184,14 @@ func (suite *TestSetupAbstractionSuite) TestListFilesInRootDir() {
 	files, err := suite.testSetup.ListFiles("/")
 	suite.NoError(err)
 	suite.Equal(1, len(files))
-	suite.Contains(files, "EXAClusterOS")
+	suite.Contains(files, "EXAClusterOS/")
 }
 
 func (suite *TestSetupAbstractionSuite) TestListFilesNonExistingDirectory() {
 	_, err := suite.testSetup.ListFiles("non-existing-dir")
-	suite.ErrorContains(err, "request failed with status 500 (500 Server Error). Response: \"E-BFSJ-11: Unable to list contents of 'non-existing-dir' in bucket bfsdefault/default: No such file or directory.\"")
+	suite.ErrorContains(err, "request failed with status 500 (500 Server Error).")
+	suite.ErrorContains(err, "E-BFSJ-11: Unable to list contents of 'non-existing-dir' in bucket 'http")
```

Comment:
split expected error message because it now contains the host name which may be different on machines